plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -201,39 +212,54 @@ export const WorkflowInstancePage = ({
               {!canRerun && (
                 <>
                   <Grid item>
-                    <Button
-                      variant="contained"
-                      color="primary"
-                      disabled={!canAbort}
-                      onClick={canAbort ? handleRerun : undefined}
+                    <Tooltip
+                      title="user not authorized to execute workflow"
+                      disableHoverListener={permittedToExecute.allowed}
                     >
-                      Retrigger
-                    </Button>
+                      <Button
+                        variant="contained"
+                        color="primary"
+                        disabled={!permittedToExecute.allowed || !canRerun}
+                        onClick={canRerun ? handleRerun : undefined}
```

Comment:
not sure about the permission change, but for me, rerun == execution, not abort

Reply:
makes sense

Reply:
I confused it with something I did in the permission check. Anyhow it is closer to execute, right, but a bit more than that -  a rerun is a call to get the parameters of a certain execution and then an execute with that. 
+1